chore: drop support for Windows 7 / 8 / 8.1 (#36427)
electron/electron: docs/breaking-changes.md

## Planned Breaking API Changes (23.0)

> ### Removed: Windows 7 and 8 support
> 
> Windows 7 and 8 are not supported anymore as Chromium 110 dropped support:
> https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en
> 

Reviewer: ```suggestion
### Removed: Windows 7 / 8 / 8.1 support

Windows 7, Windows 8, and Windows 8.1 are not supported anymore as Chromium 110 dropped support:
https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en

Older versions of Electron will continue to run on these operating systems, but Windows 10 or later will be required to run Electron v23.0.0 and higher.

```

I think we should add more details to the text to align with what Chrome is saying in their post. Also highlight 8.1 since I think it's technically a different OS from Windows 8.

## Planned Breaking API Changes (23.0)

### Removed: Windows 7 / 8 / 8.1 support

> Windows 7, Windows 8, and Windows 8.1 are no longer supported. Electron follows the planned Chromium deprecation policy, which will [deprecate Windows 7 support beginning in Chromium 109](https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en).

Reviewer 2: ```suggestion
[Windows 7, Windows 8, and Windows 8.1 are no longer supported](https://www.electronjs.org/blog/windows-7-to-8-1-deprecation-notice). Electron follows the planned Chromium deprecation policy, which will [deprecate Windows 7 support beginning in Chromium 109](https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en).
```

## Planned Breaking API Changes (23.0)

### Removed: Windows 7 / 8 / 8.1 support

> Windows 7, Windows 8, and Windows 8.1 are not supported anymore as Chromium 110 dropped support:

Reviewer 3: Super nit, just cleaning up phrasing to match the blog post

```suggestion
Windows 7, Windows 8, and Windows 8.1 are no longer supported. Electron follows the planned Chromium deprecation policy, which will [deprecate Windows 7 support beginning in Chromium 109](https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023?hl=en).
```
